HED-schema.mediawiki

```diff
@@ -195,10 +196,6 @@ From version 2.2, HED adheres to http://semver.org/ versioning.
 *** Yawn 
 ** Nonvocal <nowiki> [A car engine or gears grinding --- anything that is not made by a human or an animal]</nowiki> 
 *** Engine 
-* Olfactory <nowiki>[Odor]</nowiki> 
-* Taste 
-* Tactile <nowiki>[Pressure]</nowiki> 
-* Visual  
 ** Rendering type <nowiki>{requireChild, predicateType=passThrough}</nowiki> 
 *** Screen 
 **** Head-mounted display
```

Comment:
These should be removed as they are being moved under Attribute

Reply:
Modified pull request.